src/CalendarApp/CalendarApp.js

```diff
@@ -31,6 +31,48 @@ export default function CalendarApp() {
           />
         </main>
       </div>
+
+      <div className="calendarWorkoutContainer">
+        <div className="calendarWorkoutContainerContent">
+          <h3 className="calendarWorkoutTitle">Full Body Workout</h3>
+          <button className="calendarWorkoutButton">Start</button>
+
+          <div className="calendarExerciseList">
+            <div className="calendarExerciseItem">
```

Comment:
You can create a new component for example <calendarExerciseItem exercice="push-up" reps="20" />
and inside that component to render the items. this way you can reuse the same component and later we can map and render multiple elements with less code.

Reply:
Same point. Please add specificity to CSS file over js file. Thank you

Reply:
solid feedback, thanks